task_helper/envoy_manager.py

```diff
@@ -0,0 +1,15 @@
+import requests
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+class EnvoyManager:
+    def __init__(self) -> None:
+        return None
+
+    def healthcheck_fail(self):
+        LOGGER.info("Signalling envoy to fail the task")
+        url = "http://127.0.0.1:9901/healthcheck/fail"
+        x = requests.post(url)
```

Comment:
Is it worth wrapping this in a try / catch to enable graceful handling and logging of errors? :)